taxi/forms.py

```diff
@@ -0,0 +1,46 @@
+from django import forms
+from django.contrib.auth import get_user_model
+from django.contrib.auth.forms import UserCreationForm
+
+from taxi.models import Driver, Car
+from taxi.validators import license_number_validator
+
+
+class DriverUserCreationForm(UserCreationForm):
+    first_name = forms.CharField(max_length=256, required=True)
+    last_name = forms.CharField(max_length=256, required=True, )
+    license_number = forms.CharField(
+        required=True,
+        validators=license_number_validator
+    )
+
+    class Meta(UserCreationForm.Meta):
+        model = Driver
+        fields = UserCreationForm.Meta.fields + \
+            ("first_name", "last_name", "license_number")
```

Comment:
Use `()` instead of `\`
```suggestion
        fields = (UserCreationForm.Meta.fields + 
            ("first_name", "last_name", "license_number"))
```

Reply:
flake8 marks this change as W504
![2023 11 16 11-31-13](https://github.com/mate-academy/py-taxi-service-user-form-and-validation/assets/65740527/c2bde1df-32ee-4aa8-9ce5-6eca15f7e678)

Reply:
![Uploading 2023.11.16 11-34-41.jpg…]()

Reply:
![2023 11 16 11-34-41](https://github.com/mate-academy/py-taxi-service-user-form-and-validation/assets/65740527/ff60a72f-8df0-4dd9-aaa3-5fd329c16643)